std-bits/table/src/main/java/org/enso/table/data/column/operation/map/SpecializedIsInOp.java

```diff
@@ -55,16 +55,21 @@ public Storage<?> runMap(S storage, List<?> arg) {
     Context context = Context.getCurrent();
     CompactRepresentation<T> compactRepresentation = prepareList(arg);
     BitSet newVals = new BitSet();
+    BitSet missing = new BitSet();
     for (int i = 0; i < storage.size(); i++) {
-      if (storage.isNa(i) && compactRepresentation.hasNulls) {
-        newVals.set(i);
+      if (storage.isNa(i)) {
+        missing.set(i);
```

Comment:
What if `arg.size() == 0`? Looking at the `contains_relational` semantics, it should result in `False`, but looking at this code it seems that it will result in `Nothing`.

Do we have a test case for such scenario? If not, please let's add it and ensure the implementation is correct.

Reply:
Done.